ffi/firewood.go

```diff
@@ -126,42 +186,46 @@ const (
 	invalidCacheStrategy
 )
 
-// New opens or creates a new Firewood database with the given configuration. If
-// a nil config is provided, [DefaultConfig] will be used instead.
+// New opens or creates a new Firewood database with the given options.
 // The database file will be created at the provided file path if it does not
 // already exist.
 //
+// If no configuration options are provided, sensible defaults will be used.
```

Comment:
```suggestion
// If no [Option] is provided, sensible defaults will be used.
```
??? Not totally sure if this is an improvement. But there is a type associated with this, and it's not actually linked anywhere in this comment